gslib/commands/config.py

```diff
@@ -250,6 +250,8 @@
       tab_completion_time_logs
       tab_completion_timeout
       task_estimation_threshold
+      test_cmd_regional_bucket_location
```

Comment:
Are these related to IAM Condition?

Reply:
Sort of :)

The Test environment I was using doesn't have the `us-central1` region, so I added this flag to override the default region we use.  And I'm unable to test against the usual environment because of whitelists.  So since this was necessary for testing, yes :upside_down_face:

I then also realized that we weren't documenting `test_notification_url` either, so I documented that and added it into the commit too.

Reply:
Thanks for clarification.